pkgs/by-name/pi/pinact/package.nix

```diff
@@ -13,16 +14,26 @@ let
 in
 buildGoModule (finalAttrs: {
   pname = "pinact";
-  version = "3.4.5";
+  version = "3.6.0";
```

Comment:
This version update is outdated because PR #475447 was merged. However, we still need the Go version sync for future updates.